samples/wifi/sr_coex/README.rst

```diff
@@ -7,7 +7,7 @@ Wi-Fi: Bluetooth LE coexistence
    :local:
    :depth: 2
 
-This sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
+The Bluetooth LE coexistence sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
```

Comment:
```suggestion
The Bluetooth® Low Energy (LE) coexistence sample demonstrates coexistence between Wi-Fi® and Short Range (SR) radios in 2.4 GHz frequency.
```

Reply:
Additionally, on line 23 the trademark for Bluetooth can be removed

Reply:
I thought of this change but then I think this involves the Title change of the sample as well hence dropped it.